domain-cc/cc-app/src/python_src/api.py

```diff
@@ -47,35 +50,51 @@ def get_health_status():
     return {"status": "ok"}
 
 
+def log_lookup_table_match(
+    classification_code: int, is_in_dropdown: bool, contention_text: str
+):
+    log_contention_text = contention_text if is_in_dropdown else "Not in dropdown"
+
+    if classification_code:
+        already_mapped_text = (  # being explicit, do not leak PII
+            contention_text.strip().lower()
+        )
+        log_as_json({"Lookup table match": already_mapped_text})
+    elif is_in_dropdown:
+        log_as_json({"No lookup table match for dropdown entry": log_contention_text})
+    else:
+        log_as_json("No Lookup table match for free text")
+
+
+def log_as_json(obj: Union[dict, str]):
+    try:
+        obj["date"] = time.strftime("%Y-%m-%d %H:%M:%S", time.localtime())
+        obj["level"] = logging.getLevelName(logging.root.level)
+        logging.info(json.dumps(obj))
+    except TypeError:
+        logging.info(obj)
+
+
 @app.post("/classifier")
 def get_classification(claim: Claim) -> Optional[PredictedClassification]:
-    logging.info(
-        f"claim_id: {claim.claim_id}, form526_submission_id: {claim.form526_submission_id}"
+    log_as_json(
+        {
+            "claim_id": claim.claim_id,
+            "form526_submission_id": claim.form526_submission_id,
+        }
     )
     classification_code = None
     if claim.claim_type == "claim_for_increase":
-        logging.info(f"diagnostic code: {claim.diagnostic_code}")
+        log_as_json({"diagnostic code": claim.diagnostic_code})
         classification_code = dc_lookup_table.get(claim.diagnostic_code, None)
 
     if claim.contention_text and not classification_code:
         classification_code = dropdown_lookup_table.get(claim.contention_text, None)
         is_in_dropdown = claim.contention_text.strip().lower() in dropdown_values
-        log_contention_text = (
-            claim.contention_text if is_in_dropdown else "Not in dropdown"
+        log_as_json({"In Dropdown": is_in_dropdown})
```

Comment:
```suggestion
        log_as_json({"is_in_dropdown": is_in_dropdown})
```
Maybe keep the log field names and variable names consistent. That way it's a little easier to see how the values map to source without having to scan through too much code.

Reply:
You might also consider moving this into `log_lookup_table_match`.